cmd/archive/unpack.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"syscall"
+
+	"github.com/vmware/vic/lib/archive"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+func main() {
+	ctx := context.Background()
+	op := trace.NewOperation(ctx, "Unpack") // TODO op ID?
+	op.Debugf("New unpack operation created")
+
+	if len(os.Args) != 4 {
+		op.Errorf("Wrong number of arguments passed to unpack binary")
+		os.Exit(5)
```

Comment:
I would give these exit codes constants, defined above.  That will allow others to easily understand the return results.

const (
    InvalidArguments = iota
    InvalidFilterSpec
    ...
)

Make sure to capitalize the first letter so other code can refer to it.

Reply:
sure thing